webapp/packages/plugin-data-viewer-conditional-formatting/src/formatting/getValueForRuleParameter.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import type { IFormatRuleParameter } from './IFormatRule.js';
+
+export function getValueForRuleParameter<T extends string | boolean | number>(
+  state: Record<string, any>,
+  parameters: Record<string, IFormatRuleParameter> | undefined,
+  key: string,
+  fallback: T,
+): T;
+export function getValueForRuleParameter<T extends string | boolean | number>(
+  state: Record<string, any>,
+  parameters: Record<string, IFormatRuleParameter> | undefined,
+  key: string,
+): T | undefined;
+export function getValueForRuleParameter<T extends string | boolean | number>(
+  state: Record<string, any>,
+  parameters: Record<string, IFormatRuleParameter> | undefined,
+  key: string,
+  fallback?: T,
+): T | undefined {
```

Comment:
Do we really need overloads here? Maybe we can use fallback logic in place where we execute it